src/data/finset/basic.lean

```diff
@@ -432,6 +432,11 @@ mem_union.2 $ or.inl h
 theorem mem_union_right {a : α} {s₂ : finset α} (s₁ : finset α) (h : a ∈ s₂) : a ∈ s₁ ∪ s₂ :=
 mem_union.2 $ or.inr h
 
+theorem forall_mem_union [decidable_eq α] {s₁ s₂ : finset α} {p : α → Prop} :
```

Comment:
This is inside `section decidable_eq` which already includes this instance as a variable.
```suggestion
theorem forall_mem_union {s₁ s₂ : finset α} {p : α → Prop} :
```